config/logging.go

```diff
@@ -0,0 +1,41 @@
+package config
+
+// Logging configures overall and logger-specific log levels. Level values are
+// case-insensitive and include the following in order of importance: "fatal",
+// "panic", "dpanic", ""error", "warn", "info", "debug"
+type Logging struct {
+	// Level sets the log level for all loggers that do not have a setting in
+	// Loggers. The default value is "info".
+	Level string
```

Comment:
I'm curious why make an exceptional case for logging subsystem `*` here? Why not add a default of `info` for `*` subsystem in the map below and let the user override it like any other subystem?

Reply:
That would also work, but using "*" may imply globbing or regex name matching, which I do not want to do (see next comment).  Do you think that without globbing that using "*" would still be better?

Reply:
I have no preference. Both work :)